scripts/process_jartic_parallel.py

```
@@ -0,0 +1,320 @@
+    
+    if output_file.exists():
+        size_mb = output_file.stat().st_size / (1024 * 1024)
+        logger.info(f"Output size: {size_mb:.2f} MB")
+    
+    if failed_prefectures:
+        logger.warning(f"Failed prefectures ({len(failed_prefectures)}): {', '.join(failed_prefectures[:5])}")
+    
+    return total_records
+
+
+def sample_archive(archive_path: Path):
```

Ruff (0.12.2)

Missing return type annotation for public function `sample_archive`

Add return type annotation: `None`

(ANN201)